JitsiConference.js

```diff
@@ -458,6 +472,9 @@ JitsiConference.prototype.addTrack = function (track) {
                 this.room.setVideoMute(track.isMuted());
             }
 
+            if (this.talkMutedDetection && track.isAudioTrack())
+                this.talkMutedDetection.addTrack(track);
+
```

Comment:
I don't like that because it gets us further from the goal of having `TalkMutedDetection` as a black box. Since `this.talkMutedDetection` is already listening to `TRACK_MUTE_CHANGED`, cannot it listen to `TRACK_ADDED` instead of the above?